### ISSUE TITLE
Look into a system for pooled bounties

### ISSUE THREAD

**Issue by larsiusprime** [larsiusprime/larsBounties#3]
Sometimes people want to contribute to a bounty and it'd be nice if there were an easy coordinated way to do that which doesn't involve lots of money going to fees unnecessarily.

**Comment by kevinresol** [comment]
If we can set up something, we might name it as "Haxe Bounties" so the whole Haxe ecosystem can benefit from it.

**Comment by larsiusprime** [comment]
That's true. I guess we should investigate whether the @HaxeFoundation is interested in facilitating this somewhat (I would be happy to spearhead on a volunteer basis). Thoughts, @Merelleya ?

On the other hand, if going that way is too much overhead and slows things down, we could just keep things simple.
